test/unit/iotaRedirections-test.js

```diff
@@ -273,6 +273,39 @@ describe('IoTA Redirections', function() {
         });
     });
 
+    describe('When the creation of multiple devices arrives to the manager', function() {
```

Comment:
In case this is used as documentation ...
  multiple devices arrives => multiple devices arrive